.appveyor.yml

```diff
@@ -80,7 +80,7 @@ after_test:
 on_finish:
   - ps: |
       $url = "https://ci.appveyor.com/api/testresults/junit/$env:APPVEYOR_JOB_ID"
-      $pattern = '**\build\test-results\**\TEST-*.xml'
+      $pattern = '**\build\test-results\**\flattened\TEST-*.xml'
```

Comment:
Could we maybe get before / after links to according AppVeyor / Azure test pages in the commit message, so we can quickly see the impact of this change?

Reply:
For Azure you can compare these two builds, select "Outcome -> Clear" to successful tests and search for "prefer more complete information for GitHub". In the first link the test name is missing, in the second link it is there.
https://dev.azure.com/oss-review-toolkit/ort/_build/results?buildId=1402&view=ms.vss-test-web.build-test-results-tab
https://dev.azure.com/oss-review-toolkit/ort/_build/results?buildId=1392&view=ms.vss-test-web.build-test-results-tab

The AppVeyor build did not show the correct names when I looked yesterday, and today it does not show any test results anymore. Let's see what happens in the next build. I was assuming it's build cache related, because I also did not get the flattened test results locally without the `--rerun-tasks` option.

Reply:
For AppVeyor there are no test results because all test task results are taken from the cache and therefore the "flattened" directory is empty. I would suggest we merge this nonetheless, because eventually the cache will be outdated and test results will start to show up, especially for modules changed in the PRs.